sycl/test/sub_group/common.cpp

```diff
@@ -21,7 +21,6 @@ using namespace cl::sycl;
 struct Data {
   unsigned int local_id;
   unsigned int local_range;
-  unsigned int max_local_range;
```

Comment:
I understand why the `max_local_range` test was changed as part of this PR, but I don't think we should remove it completely.  We should still check that every work-item in the sub-group reports the same `max_local_range` value.

Reply:
updated